.github/workflows/test-hw.yml

```
@@ -0,0 +1,96 @@
+# Copyright 2021, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+
+# camkes-vm-examples hardware builds and runs
+#
+# See camkes-vm-hw/builds.yml in the repo seL4/ci-actions for configs.
+
+name: camkes-vm-examples HW
```

This is the name that is shown in the GitHub UI, we should follow the pattern the other ones have (short-ish and capitalised).

Maybe:
```suggestion
name: VM Examples HW
```

---

Would just "HW" be enough with a "build" and "run" step then?

---

In the context of this repo, yes that makes sense.